plugin.js

```diff
@@ -9,16 +9,15 @@ const Watchpack = require('watchpack');
 const error = msg => console.log(chalk.bold.red(msg));
 const info = msg => console.log(chalk.bold.blue(msg));
 
-/**
- * In some cases Webpack will require the pkg entrypoint before it actually
- * exists. To mitigate that we are forcing a first compilation.
- *
- * See https://github.com/wasm-tool/wasm-pack-plugin/issues/15
- */
-let ranInitialCompilation = false;
-
 class WasmPackPlugin {
   constructor(options) {
+    /**
+     * In some cases Webpack will require the pkg entrypoint before it actually
+     * exists. To mitigate that we are forcing a first compilation.
+     *
+     * See https://github.com/wasm-tool/wasm-pack-plugin/issues/15
+     */
+    this.ranInitialCompilation = false;
```

Comment:
I would use `_ranInitialCompilation` by convention here, could you please rename it?

Reply:
Updated